keyboards/ghs/rar/info.json

```diff
@@ -0,0 +1,24 @@
+{
+    "keyboard_name": "GHS.RAR",
+    "url": "https://gonehacking.studio/",
+    "maintainer": "Gone Hacking Studio",
+    "width": 16.5,
+    "height": 6.75,
+    "layouts": {
+        "LAYOUT_ansi": {
```

Comment:
The up arrow on these layouts seems to be out of order; per the layout macro it should be between the right shift and End.

Reply:
Oops. Dunno what happened here, but I've fixed it. Thanks!